File: geometry/pythagorean_theorem/java/pythagorean_theorem.java
public class PythagoreanFormula {

    public static boolean PythagoreanValidator(double a, double b, double c){
        return Math.pow(a, 2) + Math.pow(b, 2) == Math.pow(c, 2);
    }

    public static double retrieveHypotenuse (double a, double b){
        return Math.sqrt(Math.pow(a, 2) + Math.pow(b, 2));
    }
}

File: geometry/pythagorean_theorem/javascript/pythagorean_theorem.js

/**
 * Validates pythagorean trigonometric identity
 * @param {number} a side a
 * @param {number} b side b
 * @param {number} c hypotenuse
 */
function pythagoreanValidator( a, b, c ) {
    return a * a + b * b  === c * c;
}

/**
 * Retrieves the hypotenuse, given two sides of the right triangle
 * @param {*} a side a
 * @param {*} b side b
 */
function retrieveHypotenuse( a, b ) {
    return Math.sqrt( a * a + b * b );
}
